dataprofiler/tests/profilers/profiler_options/test_profiler_options.py

```diff
@@ -505,28 +503,6 @@ def test_eq(self, *mocks):
         options.structured_options.float.precision.sample_ratio = 0.1
         self.assertNotEqual(options, options2)
 
-    def test_json_encode(self, *mocks):
```

Comment:
this was moved to `test_profiler_class_options.py` since that test class inherits `TestBaseOptions`